worker/pkg/workflows/schemainit/workflow/workflow.go

```diff
@@ -11,6 +12,7 @@ type SchemaInitRequest struct {
 	JobId         string
 	JobRunId      string
 	DestinationId string
+	IsMysql       bool // hack for now to only use reconcile schema for mysql
```

Comment:
could make this more generic if it's easy enough to something that we can opt other drivers in to easily without having to change this property. 
`UseSchemaDrift` or something like that. That way when we add PG support, this value doesnt change.

Unless the IsMysql is used elsewhere to determine something specific.